Jenkinsfile

```diff
@@ -22,7 +22,7 @@ pipeline {
                 container('git') {
                     script {
                         sh(script: "date -u +'%Y-%m-%dT%H:%M:%SZ' > build-start", returnStdout: true).trim()
-                        tag = sh(script: 'git rev-parse --short HEAD > image-tag && cat image-tag', returnStdout: true).trim()
+                        tag = sh(script: 'git rev-parse --short HEAD > image-tag && echo "$GIT_BRANCH-`cat image-tag`"', returnStdout: true).trim()
```

Comment:
I think this needs to write `$GIT_BRANCH` to the `image-tag` file,-- the step that sends the build occurrence reads that, so the tag in the occurrence won't match

Reply:
That stage might be able to use the variable instead, but the escaping gets a little hairy.

Reply:
yeah good catch, I just noticed this in my demo deployment